### PR TITLE
Redesigning branding page

### PR DESCRIPTION
**Pull request by Aliiiu**
# Branding page redesign

This PR resolves the redesigning of the branding page

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1762

### CODE REVIEW

File: src/pages/branding/index.tsx [hunk 1]
```diff
@@ -5,120 +5,51 @@ import {
...
+import {
+  alternateLogoAssets,
+  iqLogoAsset,
+  braindaoAltLogoAssets,
+  braindaoLogoAssets,
+  iqgptLogoAssets,
+} from '../../components/Branding/brandassets'
+import BrandWrapper from '@/components/Branding/BrandWrapper'
+
+const CustomListItem = ({ content }: { content: string }) => {
```
Comment (Adebesin-Cell) [inline src/pages/branding/index.tsx:24]: Would be nice to have it as ``BrandingListItem``

File: src/components/Branding/BrandWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const [currentViewedAsset, setCurrentViewedAsset] = useState<string>('')
+
+  return (
+    <Box pb={24} maxW={{ base: '90%', '2xl': '1280px' }} mx="auto">
+      <Flex flexDir="column">
+        <Heading fontWeight={'600'} fontSize={{ base: '24px', lg: '36px' }}>
+          {title}
+        </Heading>
+        <Text mt={4} fontSize={{ lg: '20px', base: 'sm' }}>
+          {description}
```
Comment (Adebesin-Cell) [inline src/components/Branding/BrandWrapper.tsx:33]: Test if there's no description to render nothing.

``
          {description && 
          <Text mt={4} fontSize={{ lg: '20px', base: 'sm' }}>
               {description} 
          </Text/>``

**Review by Adebesin-Cell — COMMENTED** [review]
Nice factoring 👏

**Comment by Adebesin-Cell** [comment]
iq-wiki-git-redesigning-branding-page-prediqt vercel app_branding(iPhone 12 Pro)
For the banner intro text, please give padding I think 20px on both sides.

**Review by Simplypheyie96 — APPROVED** [review]
Tested on all devices and it looks great!

**Comment by Aliiiu** [comment]
> Tested on all devices and it looks great!

Gracias

File: src/components/Branding/BrandWrapper.tsx (hunk 2 above)
Comment (Adebesin-Cell) [reply src/components/Branding/BrandWrapper.tsx]: Yeah, if you pass that the Text will render but with empty content. Conditions to check for ``description`` would make it render only if there's description.

**Review by Adebesin-Cell — APPROVED** [review]

**Pull request merged**